doc/7/controllers/auth/create-api-key/index.md

```diff
@@ -0,0 +1,70 @@
+---
+code: true
+type: page
+title: createApiKey
+description: Creates a new API key for the currently loggued user.
+---
+
+# createApiKey
+
+<SinceBadge version="7.1.0" />
+
+<SinceBadge version="Kuzzle 2.1.0" />
```

Comment:
Food for thoughts: I wonder if instead of this, we should link to the underlying API method instead

Reply:
I remember that we already discuss about adding a link to the original API on each SDK method.  
But why did you say that about the Since badge?

Reply:
Because we don't need it if we have links to the underlying API. Moreover, I don't like having to maintain since badges in multiple places, I'd prefer to have only to worry about those badges in the API section of the documentation when updating an API feature.